database-plugins/src/main/java/io/cdap/plugin/db/batch/source/DBSource.java

```diff
@@ -357,4 +358,17 @@ private Schema getSchema(FailureCollector collector) {
       throw collector.getOrThrowException();
     }
   }
+
+  private void emitLineage(BatchSourceContext context) {
+    Schema schema = sourceConfig.getSchema(context.getFailureCollector());
+    if (schema == null)
```

Comment:
should always have `{` after the if condition, the checkstyle will fail, make sure you run a `mvn clean package -DskipTests` before you commit the code

Reply:
Ack.